server/auth/auth.go

```diff
@@ -380,6 +380,15 @@ func (u user) GetID() string {
 	return u.ID
 }
 
+func (u user) Raw() []knox.RawPrincipal {
+	return []knox.RawPrincipal{
+		{
+			ID:   u.ID,
+			Type: "user",
```

Comment:
Call `u.Type()` here, rather than repeating the string. That way there is a single source of truth for the value of that string.